website/static/css/custom.css

```
@@ -0,0 +1,53 @@
+/* your custom css */
+
+@media only screen and (min-device-width: 360px) and (max-device-width: 736px) {
+}
+
+@media only screen and (min-width: 1024px) {
+}
+
+@media only screen and (max-width: 1023px) {
+}
+
+@media only screen and (min-width: 1400px) {
+}
+
+@media only screen and (min-width: 1500px) {
+}
+
+html {
+  overflow-y: auto; /* To always have scrollbar visible and avoid header jump, when page is loading */
```

Oh, sorry, I suggested wrong value at first. Could you update, please?

```suggestion
  overflow-y: scroll; /* To always have scrollbar visible and avoid header jump, when page is loading */
```

---

Of course! Done. 👌